user-impl/src/main/java/com/example/auction/user/impl/UserCommand.java

```diff
@@ -4,44 +4,19 @@
 import com.fasterxml.jackson.annotation.JsonCreator;
 import com.lightbend.lagom.javadsl.persistence.PersistentEntity;
 import com.lightbend.lagom.serialization.Jsonable;
+import lombok.Value;
 
 import java.util.Optional;
 
 public interface UserCommand extends Jsonable {
+	@Value
```

Comment:
It looks like some files are indenting with tab characters instead of space characters.

Here are instructions for configuring Eclipse to use spaces instead of tabs: http://stackoverflow.com/a/408449